cpp/tests/io/parquet_test.cpp

```diff
@@ -3348,7 +3348,7 @@ TEST_F(ParquetWriterTest, CheckPageRows)
   // check first page header and make sure it has only 5000 values
   int fd = open(filepath.c_str(), O_RDONLY);
   unsigned char buf[1024];
-  read(fd, buf, sizeof(buf));
+  (void)!read(fd, buf, sizeof(buf));
```

Comment:
Why the exclamation mark? Is it not enough to cast to void?

Reply:
Tried `(void)` and got the warning still.  https://stackoverflow.com/questions/7271939/warning-ignoring-return-value-of-scanf-declared-with-attribute-warn-unused-r#comment95629501_7273026